src/sentry/integrations/github/client.py

```diff
@@ -112,7 +114,18 @@ def get_token(self):
             expires_at = datetime.strptime(expires_at, '%Y-%m-%dT%H:%M:%S')
 
         if not token or expires_at < datetime.utcnow():
-            res = self.create_token()
+            try:
+                res = self.create_token()
+            except ApiError:
+                # The token could not be refreshed. This likely indicates the
+                # installation was removed but failed to be disabled through
+                # the uninstall webhook. disable the integration now.
+                #
+                # TODO(epurkhiser): When we have some kind of logging mechanism
+                # for the user to inspect for integrations, log something here.
```

Comment:
This sounds like a great improvement for integrations.

Reply:
@EvanPurkhiser would audit log not be useful here?